tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -35,7 +35,7 @@ def wait_for_status_bar_displayed(self):
         self.wait_for_element_displayed(*self._status_bar_locator)
 
     def wait_for_notification_toaster_displayed(self, timeout=10, message=None):
-        self.wait_for_element_displayed(*self._notification_toaster_locator)
+        self.wait_for_element_displayed(*self._notification_toaster_locator, timeout=timeout)
```

Comment:
`message` isn't being passed into the `wait_for_element_displayed` call, so won't do much good here. You didn't introduce that, but maybe good to fix it anyway--either remove the parameter, or pass it along.